src/helpers/filters.ts

```diff
@@ -65,8 +65,13 @@ const fileterByToAmount = (transactions: Transaction[], toAmount: string): Trans
   });
 };
 
-export const filterBySearch = (search: string, transactions: Transaction[]): Transaction[] =>
-  transactions.filter(transaction => {
+export const filterBySearch = (
+  search: string,
+  transactionNotes: Transaction[],
+  transactions: Transaction[],
+): Transaction[] => {
+  const transactionsWithNote = transactions.map((tx: Transaction) => ({ ...tx, note: transactionNotes[tx.hash] }));
+  return transactionsWithNote.filter(transaction => {
```

Comment:
![Zrzut ekranu 2020-11-23 o 11 04 17](https://user-images.githubusercontent.com/67273035/99949640-a78a0600-2d7b-11eb-8864-65085fb88b1f.png) 
to fix it move `map((tx: Transaction) => ({ ...tx, note: transactionNotes[tx.hash] }))` lower in the getSectionData function. We don't need to map notes twice